packages/@lwc/rollup-plugin/src/__tests__/index.spec.js

```diff
@@ -4,28 +4,19 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-const fs = require('fs');
 const path = require('path');
 const rollup = require('rollup');
 const prettier = require('prettier');
 const rollupCompat = require('rollup-plugin-compat');
 const rollupCompile = require('../index');
+require('jest-utils-lwc-internals');
 
 function pretty(str) {
```

Comment:
I don't think we need to use `prettier` now that we are using snapshots.

Reply:
Doesn't it make snapshots easier to read?